.github/workflows/ci.yml

```diff
@@ -29,64 +28,18 @@ jobs:
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python 3.11
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.11
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip wheel
-        pip install -r requirements.txt
-        pip install -r requirements-dev.txt
-    - name: Run migrations
-      run: python manage.py migrate
-    - name: Run tests
-      run: python manage.py test -v 2
-      
-  publish:
-    runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/develop'
-    needs: test
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-      
-    - name: Login to GitHub Container Registry
-      uses: docker/login-action@v1
-      with:
-        registry: ghcr.io
-        username: ${{ github.actor }}
-        password: ${{ secrets.GITHUB_TOKEN }}
-        
-    - name: Get image metadata
-      id: meta
-      uses: docker/metadata-action@v3
-      with:
-        images: ghcr.io/${{ github.repository }}
-        
-    - name: Build and push Docker image for the web app
-      uses: docker/build-push-action@v2
-      with:
-        context: .
-        push: true
-        tags: ${{ steps.meta.outputs.tags }}
-
-  docs:
-    needs: test
-    timeout-minutes: 15
-    runs-on: ubuntu-latest
```

Comment:
It's probably useful to have a test-docs step here that runs `mkdocs build` just to make sure the docs build successfully so they don't fail on release.